### PR TITLE
Revert bundle identifiers for safari extension

### PR DESCRIPTION
**Pull request by darinwf**
Fixes release build

### CODE REVIEW

**Comment by darinwf** [comment]
This is restoring to the values from before 7ce3afbae44eef49f3ccd531d5eaeb01f4389050

File: Client.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -8869,7 +8869,7 @@
 					"-framework",
 					SafariServices,
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = "co.neeva.app.ios.browser-dev.neeva-for-safari.safariExtension";
+				PRODUCT_BUNDLE_IDENTIFIER = co.neeva.app.ios.browser.SafariExtension;
```
Comment (evancrow) [inline Client.xcodeproj/project.pbxproj:8872]: `"` shouldn't be removed

**Review by evancrow — APPROVED** [review]
LGTM - with one change

File: Client.xcodeproj/project.pbxproj (hunk 1 above)
Comment (darinwf) [reply Client.xcodeproj/project.pbxproj:8872]: interesting! i just did a revert using the diff... i wonder, maybe those quotes don't matter. i'll add them in

**Pull request merged**